api_tests/users/views/test_user_actions.py

```diff
@@ -21,6 +21,14 @@ class TestActionFilters(ActionFilterMixin):
     def url(self):
         return '/{}users/me/actions/'.format(API_BASE)
 
+    @pytest.fixture()
+    def expected_actions(self, all_actions, allowed_providers):
+        expected = super(TestActionFilters, self).expected_actions(all_actions, allowed_providers)
+        node = expected[0].target.node
+        node.is_public = False
+        node.save()
+        return [a for a in expected if a.target.node.is_public]
```

Comment:
Same here, `a` --> `action`

Reply:
I changed `expected` to `actions` to hopefully make it clearer?